### PR TITLE
Update tsh.mdx to include --speed option for tsh play

### PR DESCRIPTION
**Pull request by tenaciousdlg**
In v15 --speed was added for tsh play but is not listed in the docs. Proposing a change to get that added with a small example.

### CODE REVIEW

File: docs/pages/connect-your-client/tsh.mdx [hunk 1]
```diff
@@ -952,6 +952,12 @@ the `--format` flag of `tsh play`:
 | `json` | Servers, Kubernetes clusters, applications, databases | `tsh` prints a JSON-serialized list of audit events, separated by newlines. |
 | `yaml` | Servers, Kubernetes clusters, applications, databases | `tsh` prints a YAML-serialized list of audit events, separated by `---` characters. |
 
+The `tsh play` command can also be used to speed up recordings for SSH and Kuberntes.
```
Comment (webvictim) [inline docs/pages/connect-your-client/tsh.mdx:955]:
```suggestion
The `tsh play` command can also be used to speed up recordings for SSH and Kubernetes.
```

File: docs/pages/connect-your-client/tsh.mdx [hunk 2]
```diff
@@ -952,6 +952,12 @@ the `--format` flag of `tsh play`:
...
+
+```code
+tsh play --speed 8x UUID
```
Comment (webvictim) [inline docs/pages/connect-your-client/tsh.mdx:958]:
```suggestion
tsh play --speed 8x <session-id>
```

File: docs/pages/connect-your-client/tsh.mdx (hunk 1 above)
Comment (zmb3) [reply docs/pages/connect-your-client/tsh.mdx:955]: Ooops sorry, I meant to leave my own comment, not edit Gus's 😇

**Review by zmb3 — APPROVED** [review]

**Comment by zmb3** [comment]
I've gone ahead and applied my own suggestions since this has been sitting for over a month. @ptgott can you please review?

**Review by ptgott — APPROVED** [review]

**Pull request merged**